src/server/visual-diff-plugin.js

```diff
@@ -1,9 +1,12 @@
-import { access, constants, mkdir, readdir, rename, rm, stat } from 'node:fs/promises';
+import { access, constants, mkdir, readdir, readFile, rename, rm, writeFile } from 'node:fs/promises';
 import { basename, dirname, join } from 'node:path';
 import { env } from 'node:process';
+import pixelmatch from 'pixelmatch';
+import { PNG } from 'pngjs';
 
 const isCI = !!env['CI'];
 const DEFAULT_MARGIN = 10;
+const DEFAULT_TOLERANCE = 0; // TODO: Support tolerance override?
```

Comment:
We currently allow this to be overridden and a few places have, but I don't think we have a story for it